snowplow-tracker/src/main/java/com/snowplowanalytics/core/remoteconfiguration/ConfigurationState.java

```diff
@@ -1,4 +1,4 @@
-package com.snowplowanalytics.snowplow.internal.remoteconfiguration;
+package com.snowplowanalytics.core.remoteconfiguration;
```

Comment:
This enum is also published from the `Snowplow` class and I think it should be in the snowplow folder.